### PR TITLE
amfora: remove unnecessary postInstall phase for darwin

### PR DESCRIPTION
**Pull request by freezeboy**
###### Motivation for this change

I forgot a part of https://github.com/NixOS/nixpkgs/pull/107884#issuecomment-758553109 before it was merged

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by freezeboy** [comment]
@sikmir sorry about that

**Review by sikmir — APPROVED** [review]

File: pkgs/applications/networking/browsers/amfora/default.nix [hunk 1]
```diff
@@ -13,7 +13,7 @@ buildGoModule rec {
 
   vendorSha256 = "0mkk7xxfxxp1w9890mkmag11mzxhy2zmh8v1macpyp1zmzgs21f8";
 
-  postInstall = ''
+  postInstall = lib.optionString (!stdenv.isDarwin) ''
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/browsers/amfora/default.nix:16]:
```suggestion
  postInstall = lib.optionalString (!stdenv.isDarwin) ''
```

Comment (freezeboy) [reply pkgs/applications/networking/browsers/amfora/default.nix:16]: My bad .. fixing it

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 109142` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>amfora</li>
</ul>
</details>

**Pull request merged**